src/AppBundle/EventSubscriber/PullRequestSubscriber.php

```diff
@@ -93,6 +95,24 @@ public function initPullRequestTypeLabel(GitHubEvent $githubEvent)
         ]);
     }
 
+    /**
+     * Add BC break label to the pull request according to the PR template.
+     *
+     * @param GitHubEvent $githubEvent
+     */
+    public function initBCBreakLabel(GitHubEvent $githubEvent)
```

Comment:
```suggestion
    public function initBCBreakLabel(GitHubEvent $githubEvent): void
```

Reply:
Not applied

Reply:
I don't know what I'm doing wrong, each time I make a change, your suggestion goes away 😅
Fixing it right now!